docs/doxygen/vic_doxygen.cfg

```diff
@@ -1364,7 +1364,7 @@ DISABLE_INDEX          = NO
 # The default value is: NO.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-GENERATE_TREEVIEW      = NO
+GENERATE_TREEVIEW      = No
```

Comment:
Should these options all really be "No" instead of "NO"? In that case, shouldn't `HAVE_DOT` be "Yes" instead of "YES"? And the default values described in comments should then have matching capitalization.